backend/src/main/java/com/redhat/cloud/notifications/routers/ValidationEndpoint.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.models.EventType;
+import io.smallrye.mutiny.Uni;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/validation")
```

Comment:
This path should either start with `com.redhat.cloud.notifications.Constants.INTERNAL` if this is an internal API or `com.redhat.cloud.notifications.Constants.API_NOTIFICATIONS_V_1_0` if it's supposed to be accessed over the Internet.

FYI all internal APIs are currently regrouped in [InternalService](https://github.com/RedHatInsights/notifications-backend/blob/master/backend/src/main/java/com/redhat/cloud/notifications/routers/InternalService.java).

Reply:
I would say it's an internal one then. Do you agree @pilhuhn ?